tests/by-util/test_df.rs

```diff
@@ -204,6 +206,27 @@ fn test_exclude_type_option() {
     new_ucmd!().args(&["-x", "ext4", "-x", "ext3"]).succeeds();
 }
 
+#[test]
+fn test_exclude_all_types() {
+    let fs_types = new_ucmd!()
+        .arg("--output=fstype")
+        .succeeds()
+        .stdout_move_str();
+    let fs_types: HashSet<_> = fs_types.lines().skip(1).collect();
```

Comment:
Nice test! Is there a particular reason to use a `HashSet` here instead of a `Vec`? If not then `HashSet` is still fine, but I'm just curious :)

Reply:
No, it was more of an intuitive decision to use a `HashSet`. It avoids duplicates because for the test it only matters that a file system type is there, but not how often.